apps/web/pages/event-types/[type].tsx

```diff
@@ -1525,6 +1574,9 @@ const EventTypePage = (props: inferSSRProps<typeof getServerSideProps>) => {
                       details = { address: values.locationAddress };
                     }
 
+                    if (newLocation === LocationType.Link) {
```

Comment:
Adding the link info info the details part - This should probably be made a switch statement if we get any more cases like this